### PR TITLE
working-groups: add talent plan courses working group

### PR DESCRIPTION
**Pull request by JinLingChristopher**

### CODE REVIEW

File: working-groups/README.md [hunk 1]
```diff
@@ -18,21 +18,22 @@ See [Working Group Template](./wg-template.md) for details.
...
+- [wg-run-discourse-on-tidb](./wg-run-discourse-on-tidb.md)
+- [wg-stmt-summary](./wg-stmt-summary.md)
+- [wg-test-infra](./wg-test-infra.md)
+- [wg-drainer-relay-log](./wg-drainer-relay-log.md)
+- [wg-replace-loader-with-lightning](./wg-replace-loader-with-lightning.md)
+- [wg-pitr](./wg-pitr.md)
+- [wg-dm-portal](./wg-dm-portal.md)
+- [wg-raw-kv-backup-restore](./wg-raw-kv-backup-restore.md)
+- [wg-dumpling](./wg-dumpling.md)
+- [wg-online-restore](./wg-online-restore.md)
```
Comment (zz-jason) [inline working-groups/README.md]: please keep the old formmat

File: working-groups/wg-talent-plan-courses.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Talent Plan Courses Working Group
+
+## Goals
+
+THe Talent Plan Courses Working Group aims to develop a series of courses to help more peole enter the field of computer infrastructure system and the TiDB ecosystem.
+
+## Work and Meeting
+
+- Regular Working Group Meeting: 4pm on every Thursday.
```
Comment (gingerkidney) [inline working-groups/wg-talent-plan-courses.md]: please add time zoon

File: working-groups/wg-talent-plan-courses.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Meeting Zoom: https://pingcap.zoom.com.cn/j/9681232794
+- [Meeting Notes](https://docs.google.com/document/d/1zDUoERn1MQYMR1PX0r0SuX8zlQIaQCjS9UhsbIBnn3g/edit)
```
Comment (gingerkidney) [inline working-groups/wg-talent-plan-courses.md]: can the meeting notes open for the community?

Comment (JinLingChristopher) [reply working-groups/wg-talent-plan-courses.md]: this is still under discussion. I will alter the content.

File: working-groups/wg-talent-plan-courses.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Talent Plan Courses Working Group
+
+## Goals
+
+THe Talent Plan Courses Working Group aims to develop a series of courses to help more peole enter the field of computer infrastructure system and the TiDB ecosystem.
```
Comment (zz-jason) [inline working-groups/wg-talent-plan-courses.md]:
```suggestion
The Talent Plan Course Working Group aims to develop a series of courses to help people enter the field of computer infrastructure system and TiDB ecosystem.
```

Comment (zz-jason) [reply working-groups/wg-talent-plan-courses.md]: This goal is not concert. Please describe more about:
1. what courses are to be developed.
2. the RFC for talent plan courses.

File: working-groups/wg-talent-plan-courses.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## Organizer
+
+- [Soline324](https://github.com/Soline324), [PingCAP](https://github.com/pingcap)
+- [JinLingChristopher](https://github.com/JinLingChristopher), [PingCAP](https://github.com/pingcap)
+
+## Member
+
+- [lamxTyler](https://github.com/lamxTyler), [PingCAP](https://github.com/pingcap)
+- [time-and-fate](https://github.com/time-and-fate)
```
Comment (zz-jason) [inline working-groups/wg-talent-plan-courses.md]: member list are WIP.

Comment (JinLingChristopher) [reply working-groups/wg-talent-plan-courses.md]: ok, I will alter it.

File: working-groups/wg-talent-plan-courses.md (hunk 2 above)
Comment (zhangjinpeng1987) [reply working-groups/wg-talent-plan-courses.md]: zone

File: working-groups/wg-talent-plan-courses.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+* [Soline324](https://github.com/Soline324), [PingCAP](https://github.com/pingcap)
+* [JinLingChristopher](https://github.com/JinLingChristopher), [PingCAP](https://github.com/pingcap)
+
+## Member (WIP)
+
+to be determined
+
+## Contact
+
+* Slack: channel **#wg-talent-plan-courses** in the [tidbcommunity](https://app.slack.com/client/TH91JCS4W/CR746G049) slack workspace.
```
Comment (zz-jason) [inline working-groups/wg-talent-plan-courses.md]:
```suggestion
* Slack: channel **#wg-talent-plan-courses** in the [tidbcommunity](https://pingcap.com/tidbslack/) slack workspace.
```

**Pull request merged**